package.json

```diff
@@ -26,12 +26,11 @@
     "jest:14": "run-s \"jest {@}\" --",
     "jest:16": "run-s \"jest {@}\" --",
     "jest-debug": "cross-env NODE_OPTIONS=--inspect-brk yarn jest --testTimeout=100000000",
-    "lint": "run-s ls-lint eslint prettier markdown-lint git-check doc-fence-check",
+    "lint": "run-s ls-lint type-check eslint prettier markdown-lint git-check doc-fence-check",
```

Comment:
I always forget `type-checks` locally because of it being outside of `lint`. I think most people would expect to have it here.